plugins/parsers/csv/parser.go

```diff
@@ -262,6 +383,11 @@ outer:
 		tags[k] = v
 	}
 
+	// add default fields
+	for k, v := range p.DefaultFields {
+		recordFields[k] = v
+	}
```

Comment:
So if a field with the same name would already exist, then you just override it with data from the file's metadata?

Reply:
Good point @Hipska. But this is the same as tags too. If you have global tags and the same column tags exist, the global tags would overwrite the column tags.


I could move both to the top but that would be a big change. 

I think this should be addressed on a different task.

Reply:
You are right, that's indeed how it already works for global tags. But here we are talking about fields, and we can keep the current implementation, but I would log a warning (or lower severity) message stating that the field has been overwritten. So the user knows how it comes the field value seems to be the same for every row in the csv.

Reply:
Very good point and idea.

Reply:
Could it be you didn't push this change yet?